package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-spellchecker",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
```suggestion
  "version": "2.1.0",
```

Let's leave this at the current version, we'll bump this right after!

Reply:
done.